src/requests/getImages.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+const getImages = (query) => {
+  if (!query) {
+    return Promise.resolve([]);
+  } else {
+    return axios
+      .get(`https://images-api.nasa.gov/search?q=${query}`)
+      .then((response) => {
+        const imageResults = response.data.collection.items
+
+        const parsedImages = imageResults.filter(img =>
+          img.data[0].media_type === "image"
+          );
+
+        const images = parsedImages.map(image => image.links[0].href
```

Comment:
Do you want to try rewriting this map and filter as a reduce? Just to keep your skills sharp 👍